merlin-server/constraints-dsl-compiler/src/libs/constraints-edsl-fluent-api.ts

```diff
@@ -73,6 +51,26 @@ export class Windows {
     this.__astNode = expression;
   }
 
+  /**
+   * Produces windows for each activity present in the plan and belonging to one of the activity types passed
+   *
+   * @param activityTypes the activity types
+   */
+  public static duringAll<A extends Gen.ActivityType>(list: Gen.ActivityType[]) : Windows {
+    return Windows.All(
```

Comment:
I think you intended to use Windows.Any? `All` is intersection/and, `Any` is union/or.

I've realized recently that the names `All` and `Any` are kinda vague. I plan to open a bikeshedding PR this week that changes these names back to `And` and `Or`, like they were before I came along and thought I knew better ;)